### PR TITLE
[IMP] various: adapt 12.0 to work with ubuntu jammy (22.04)

### PR DESCRIPTION
**Pull request by d-fence**

### CODE REVIEW

File: addons/website/models/res_partner.py [hunk 1]
```diff
@@ -28,15 +24,15 @@ def google_map_img(self, zoom=8, width=298, height=298):
...
+        return '//maps.googleapis.com/maps/api/staticmap?'+werkzeug.urls.url_encode(params)
 
     @api.multi
     def google_map_link(self, zoom=10):
         params = {
             'q': '%s, %s %s, %s' % (self.street or '', self.city or '', self.zip or '', self.country_id and self.country_id.name_get()[0][1] or ''),
             'z': zoom,
         }
-        return urlplus('https://maps.google.com/maps', params)
+        return 'https://maps.google.com/maps' + werkzeug.urls.url_encode(params)
```
Comment (Julien00859) [inline addons/website/models/res_partner.py:35]: shouldn't it be `werkzeug.urls.url_encode_plus` instead?

**Review by Julien00859 — COMMENTED** [review]
* werkzeug -> werkzeug.urls :+1: 
* decodestring -> decodebytes :+1: 
* various werkzeug things :+1:
* threading isAlive -> is_alive :+1:
* lxml get_children -> `__iter__` :+1:
* collections.Mapping -> collections.abc.Mapping :+1:

I didn't review:
* the various VAT thingies
* tests/common.py
* safe_eval
* the various requements.txt and alike files

File: addons/website/models/res_partner.py (hunk 1 above)
Comment (d-fence) [reply addons/website/models/res_partner.py:35]: > shouldn't it be `werkzeug.urls.url_encode_plus` instead?

It doesn't exists AFAIK

**Comment by d-fence** [comment]
@fw-bot ignore

**Comment by d-fence** [comment]
robodoo r+

**Comment by d-fence** [comment]
robodoo rebase-ff

**Comment by nim-odoo** [comment]
As expected, this was trivial.

**Pull request closed without merging**